### ISSUE TITLE
suddenly not working

### ISSUE THREAD

**Issue by IsaiahJean** [udacity/frontend-grading-engine#44]
The extension was working for me two days ago and now it wont work. I've followed the instructions same as I did the first time, but it still wont work.

**Comment by michaeljota** [comment]
This is happen to me too. I disallow the extension for `localhost` domain, but now I'm unable to use it as it seems like its frozen.

**Comment by IsaiahJean** [comment]
Yeah, we have to store the html file in a local drive for the extension to work. I was  placing it in an external file drive.

**Comment by schreiber2011** [comment]
Do I have to add localhost on options? Cause I tried localhost and path and it is not working.
